internal/kclvm_py/scripts/update-kclvm.sh

```diff
@@ -57,22 +53,22 @@ cp ./target/release/kclvm_cli $kclvm_install_dir/bin/kclvm_cli
 # libkclvm_cli
 
 # Darwin dylib
-if [ -e $topdir/kclvm/target/release/libkclvm_cli_cdylib.dylib ]; then
+if [ -e target/release/libkclvm_cli_cdylib.dylib ]; then
```

Comment:
In addition, we can consider the complete installation of `python3 - m pip install kclvm` (including kclvm rust core libs 
 e.g., libkclvm.dylib).

Reply:
> I suggest that the copy of the library related to "Rust" can be placed in `kclvm/build.rs`. The conditional compilation feature can be used to avoid repeated copying.

add todo comment